generation/types/main.go

```diff
@@ -289,6 +291,14 @@ func Types(field Descriptor, root *descriptorpb.FileDescriptorProto) (GoType, Gq
 	if field.GetTypeName() == ".graphql.PageInfo" {
 		return "pg.PageInfo", "pg.PageInfo", Object
 	}
+
+	for pkg, _ := range packageImportMap {
+		typeName := field.GetTypeName()[1:]
+		if strings.HasPrefix(typeName, pkg) {
+			return GoType(typeName), GqlType(typeName), Common
```

Comment:
I'm assuming here the common package name will be the same in the proto as in go, think this is fine for now

Reply:
Not sure exactly how go packages work, if a file is in `github.com/kitt-technology/protoc-gen-graphql/foo` does that guarantee the go package name is foo?